### PR TITLE
feat: remove tracking contributors from new workspace page

### LINKED ISSUE #2771: Feature: Remove "Tracked contributors" section from the "new workspace" page
### Type of feature

🍕 Feature

### Current behavior

The tracked contributors table is still present in the "create new workspace" page, but we have removed that flow from the feature.
image

### Suggested solution

Remove the tracked contributors section, the same way we did for the workspace settings page 

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

We've decide to sunset this feature for the moment.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Closes #2771

## Mobile & Desktop Screenshots/Recordings

**Before**

CleanShot 2024-02-26 at 16 05 46

**After**

CleanShot 2024-02-26 at 16 05 04

### New Workspace

1. Ensure you're logged in
2. Create a new worskpace by going to `/workspaces/new`
3. Notice there is no option in the settings to add tracked contributors.
4. Add a name for the workspace and some tracked repositories (or not)
5. Create the workspace
6. It's created successfully.

### Existing Workspace

1. Ensure you're logged in
2. Go to an existing workspace settings, e.g. `/workspaces/some-id/settings`
3. Notice there is no option in the settings to add tracked contributors.
4. Add a name for the workspace and some tracked repositories (or not)
5. Update the workspace
6. It's updated successfully.

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: lib/utils/workspace-utils.ts [hunk 1]
```diff
@@ -7,19 +7,19 @@ export async function createWorkspace({
...
   members: any[];
   sessionToken: string;
   repos: { full_name: string }[];
-  contributors: { login: string }[];
 }) {
   const { data, error } = await fetchApiData<Workspace>({
     path: "workspaces",
     method: "POST",
-    body: { name, description, members, repos, contributors },
+    // We're not tracking contributors at the moment, but for the time
```
[the remark this answers is not in the document]
Comment (jpmcb) [reply lib/utils/workspace-utils.ts:20]: `contributors` is more or less optional in the API and sending an empty array sounds like the right solution.

**Review by jpmcb — COMMENTED** [review]
Are we completely abandoning this feature? Or we anticipate bringing this back eventually? If so, I'd advocate for not removing the code completely but leaving it as "orphaned" until the time is right.

For the API, if we're eventually bringing this back, I'd also advocate leaving those routes as is.

**Comment by brandonroberts** [comment]
It's not being completely abandoned, just not shipping it for the MVP

**Comment by nickytonline** [comment]
> It's not being completely abandoned, just not shipping it for the MVP

I think I misunderstood the mission. 😅 I've reverted and pushed only the necessary changes to temporarily remove this feature of workspaces.

**Review by brandonroberts — APPROVED** [review]

**Review by jpmcb — APPROVED** [review]

**Pull request merged**